source/features/new-repo-disable-projects-and-wikis.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL} from '../github-helpers';
+
+async function disableWikiAndProjects(): Promise<void> {
+	delete sessionStorage.rghNewRepo;
+
+	void api.v3(`repos/${getRepoURL()}`, {
+		method: 'PATCH',
+		body: {
+			has_projects: false,
+			has_wiki: false
+		}
+	});
+
+	const wiki = await elementReady('[data-tab-item="wiki-tab"]');
+	wiki?.closest('.d-flex')!.remove();
+	const projects = await elementReady('[data-content="Projects"]');
+	projects?.closest('.d-flex')!.remove();
```

Comment:
These also exist in the dropdown.

Also why the `?`?
It should be a `!`

Reply:
I was having the sometimes the projects tab did not load after repo creation. So I `?` them all

Reply:
I suggest a couple more changes:

- add `await domLoaded` before this instead of using elementReady; in this case it feels ok to wait a second longer;
- you could even `await` the API call to make an error slightly more visible
- do you want to look into using XPath? Looks like a good opportunity here 
    - https://devhints.io/xpath

Reply:
> * you could even `await` the API call to make an error slightly more visible
Adding it to the the other 3 errors 😄 

But that makes sense since the token may not have the correct permissions

> do you want to look into using XPath? Looks like a good opportunity here
> * [devhints.io/xpath](https://devhints.io/xpath)

I can look into, but will have more time to explore next week sometime

Reply:
"//*[@data-content='Projects']/ancestor-or-self::*[@class='d-flex']", "//*[@data-content='Wiki']/ancestor-or-self::*[@class='d-flex']"

Reply:
> These also exist in the dropdown.

This still applies

Reply:
Which drop down are you seeing it in? 

![image](https://user-images.githubusercontent.com/16872793/92925329-58813580-f408-11ea-8a11-011115b4b739.png)

After I removed them I dont see it in the drop down

Reply:
Look in the dom.

<img width="563" alt="" src="https://user-images.githubusercontent.com/1402241/92978790-6f4e7900-f456-11ea-97b9-a2193e16a752.png">

Reply:
I am not seeing this on my end. Very weird. I will post a screen show tomorrow.
Side note: Sorry for the earlier screenshot I pasted the wrong one.

Reply:
![image](https://user-images.githubusercontent.com/16872793/93019579-ed6d6580-f5a5-11ea-8d48-bfbaa7ca91ca.png)


After I removed the tab it no longer is in the drop down.

Reply:
LOOK
IN
THE
DOM

If you remove the tab, GitHub won’t un-hide it from the dropdown, but it’s still there, hidden. If the window is small to begin with, there’s a chance that it appears in the dropdown.

Reply:
I saw the dom...
> If the window is small to begin with

...Got it